.github/workflows/pr-preview.yml

```
@@ -0,0 +1,19 @@
+  deploy-preview:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Deploy Preview App
+        uses: superfly/fly-pr-review-apps@1.0.0
+        with:
+          name: pr-${{ github.event.number }}-${{ github.repository }}
+          org: personal
+          region: syd
+        env:
+          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }} 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)